src/projects/apps/designer/margin_border_padding.md

```diff
@@ -0,0 +1,41 @@
+# Margin & Border & Padding 
```

Comment:
While it makes sense for this to be a single file from the authoring perspective, it's not so good from the UI perspective. Basically where does the button go? Margin, padding and border are 3 different panels